### PR TITLE
Pass KVM device to docker

### PR DESCRIPTION
**Pull request by okartau**
Passing /dev/kvm into docker session.
This enables running qemu with KVM mode enabled.

### CODE REVIEW

File: docker/test-qemu.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+testfile_dir = WORKSPACE + "/build/tmp-glibc/deploy/testsuite/refkit-image-common"
+qemu_args = WORKSPACE+"/build/tmp-glibc/work/intel_corei7_64-refkit-linux/refkit-image-common/1.0-r0/recipe-sysroot-native/usr/bin/qemu-system-x86_64 " \
+        "-device virtio-net-pci,netdev=net0,mac=52:54:00:12:35:02 -netdev " \
+        "user,id=net0,hostfwd=tcp::2222-:22,hostfwd=tcp::2323-:23 " \
+        "-drive if=virtio,file="+WORKSPACE+"/build/tmp-glibc/deploy/images/intel-corei7-64/refkit-image-common-intel-corei7-64.wic,format=raw " \
+        "-no-reboot -vga vmware -show-cursor -usb -usbdevice tablet " \
+        "-device virtio-rng-pci -drive if=pflash,format=qcow2,file="+WORKSPACE+"/build/tmp-glibc/deploy/images/intel-corei7-64/ovmf.qcow2 " \
+        "-enable-kvm " \
+        "-vga std -cpu Nehalem -m 256 -serial mon:vc -serial null"
+runtest_args = "iottest/runtest.py " \
```
Comment (rojkov) [inline docker/test-qemu.py]: This part would be more readable if formatted as
```
qemu_args = """ \
    %(workspace)/build/tmp-glibc/work/intel_corei7_64-refkit-linux/refkit-image-common/1.0-r0/recipe-sysroot-native/usr/bin/qemu-system-x86_64 \
    ...
    -vga std -cpu Nehalem -m 256 -serial mon:vc -serial null""" % {"workspace": os.environ["WORKSPACE"]}
```

File: docker/test-qemu.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        f.write("  Fail:"+str(failed_count)+"  Skip:"+str(skipped_count))
+        f.write("  Error:"+str(errors_count)+"\n")
+        if total_count > 0:
+            run_rate = 100*total_count/tests_count
+            pass_rate_of_total = 100*passed_count/tests_count
+            pass_rate_of_exec = 100*passed_count/total_count
+            f.write("  Run rate:" + str(run_rate) + "%")
+            f.write("  Pass rate of total:" + str(pass_rate_of_total) + "%")
+            f.write("  Pass rate of exec:" + str(pass_rate_of_exec) + "%\n")
+            f.write(67*"-" + "\n")
```
Comment (rojkov) [inline docker/test-qemu.py]: This could be 2 write syscalls instead of 9 if template strings were used.

File: docker/test-qemu.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        "-no-reboot -vga vmware -show-cursor -usb -usbdevice tablet " \
+        "-device virtio-rng-pci -drive if=pflash,format=qcow2,file="+WORKSPACE+"/build/tmp-glibc/deploy/images/intel-corei7-64/ovmf.qcow2 " \
+        "-enable-kvm " \
+        "-vga std -cpu Nehalem -m 256 -serial mon:vc -serial null"
+runtest_args = "iottest/runtest.py " \
+            "-f iottest/testplan/refkit-image-common-qemu.manifest " \
+            "-m intel-corei7-64 -t 127.0.0.1:2222 -s 10.0.2.2 --xunit ."
+
+subprocess.check_output(("tar -xzf "+ testfile_dir + "/iot-testsuite.tar.gz").split())
+subprocess.check_output(("tar -xzf "+ testfile_dir + "/iot-testfiles.intel-corei7-64.tar.gz  -C iottest/").split())
```
Comment (rojkov) [inline docker/test-qemu.py]: Generally interpreted strings (`"tar -xzf %s/iot-testsuite.tar.gz" % testfile_dir`) look more eye candy than string "summation".

File: docker/test-qemu.py (hunk 2 above)
Comment (okartau) [inline docker/test-qemu.py]: thanks, but this feedback should go to #94, where I picked few commits from.
My intention in this PR is only to get KVM mode enabled, and to verify,
I re-used #94 commits to enable some qemu launching.
I think #94 is going under re-design anyway

**Comment by okartau** [comment]
closing this as same commit was added as part of #97

**Pull request closed without merging**